core/templates/pages/story-editor-page/editor-tab/story-editor.directive.html

```diff
@@ -438,6 +439,10 @@ <h3 class="story-card-header">Story Card</h3>
       width: 100%;
     }
 
+    story-editor .content-container {
+      margin-bottom: 0
+    }
```

Comment:
Did you purposely remove the semicolon (`;`) at the end?

Reply:
> Did you purposely remove the semicolon (`;`) at the end?

No, it got removed by mistake while I was trying to check why all test cases were failing. Sorry, will add it back.